packages/image_picker/image_picker_for_web/lib/image_picker_for_web.dart

```diff
@@ -96,6 +97,17 @@ class ImagePickerPlugin extends ImagePickerPlatform {
     return _getSelectedFile(input);
   }
 
+  /// Injects a file input, and returns a list of PickedFile that the user selected locally.
+  @override
+  Future<List<PickedFile>?> pickMultiImage(
+      {double? maxWidth, double? maxHeight, int? imageQuality}) {
+    html.FileUploadInputElement input =
+        createInputElement(_kAcceptImageMimeType, null, multiple: true)
+            as html.FileUploadInputElement;
+    _injectAndActivate(input);
+    return _getSelectedFiles(input);
+  }
+
```

Comment:
This is only used by a deprecated method of the latest version of the plugin. Let's not implement it for the web. Please remove this implementation (it is also untested :P)

Reply:
removed it